packages/optimise-core/test/meddraUpload.test.js

```diff
@@ -0,0 +1,47 @@
+/* global beforeAll afterAll describe test expect */
+
+const request = require('supertest');
+const admin = request.agent(global.optimiseRouter);
+const FormData = require('form-data');
+const { connectAdmin, disconnectAgent } = require('./connection');
+
+beforeAll(async () => {
+    await connectAdmin(admin);
```

Comment:
This should also be checking that traditional users cannot upload.
```suggestion
    await connectAdmin(admin);
    await connectUser(admin);
```